plato/draw/povray/Ellipsoids.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+import rowan
+from ... import draw
+
+class Ellipsoids(draw.Ellipsoids):
+    __doc__ = draw.Ellipsoids.__doc__
+
+    def render(self, rotation=(1, 0, 0, 0), **kwargs):
+        positions = rowan.rotate(rotation, self.positions)
+        orientations = rowan.multiply(
+            rotation, rowan.normalize(self.orientations))
+        rotations = np.degrees(rowan.to_euler(orientations))
```

Comment:
The orientations don't seem to be working correctly (compare the image in the PR description vs https://58-169864207-gh.circle-artifacts.com/0/tmp/plato_output/vispy_ellipsoids.png to see the difference); I'd guess the angles are specified in a different order for povray or something like that? Alternatively, the math for rotation matrices is unambiguous and could be copied from `ConvexPolyhedra`.

Reply:
Thanks - I looked at the images very fast and didn't see the difference at first. I think this only requires a very minor change.

Reply:
Fixed - `rowan` was computing the right values with the `zyx` convention and `intrinsic` axis type (which are the defaults), but the angles were just being fed to povray in the wrong order.